clients/client-python/gravitino/api/credential/oss_token_credential.py

```diff
@@ -0,0 +1,91 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from abc import ABC
+from typing import Dict
+
+from gravitino.api.credential.credential import Credential
+
+
+class OSSTokenCredential(Credential, ABC):
+    """Represents OSS token credential."""
+
+    OSS_TOKEN_CREDENTIAL_TYPE = "oss-token"
+    _GRAVITINO_OSS_SESSION_ACCESS_KEY_ID = "oss-access-key-id"
+    _GRAVITINO_OSS_SESSION_SECRET_ACCESS_KEY = "oss-secret-access-key"
+    _GRAVITINO_OSS_TOKEN = "oss-security-token"
+
+    def __init__(self, credential_info: Dict[str, str], expire_time_in_ms: int):
+        self._access_key_id = credential_info[self._GRAVITINO_OSS_SESSION_ACCESS_KEY_ID]
+        self._secret_access_key = credential_info[
+            self._GRAVITINO_OSS_SESSION_SECRET_ACCESS_KEY
+        ]
+        self._security_token = credential_info[self._GRAVITINO_OSS_TOKEN]
+        self._expire_time_in_ms = expire_time_in_ms
```

Comment:
Do you need to verify these variables (whether they are null or not)?

Reply:
added